wp/plugin/wp.opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
```

Comment:
This opam file misses the build and install steps, which is a bit surprising. I would suggest using dune-project to encode your dependencies. Not only it has nicer syntax, but it also properly handles plugin installations. You can use the wp.opam.template file if you need to override any fields.